locales/csv/.gitignore

```diff
@@ -0,0 +1 @@
+*.csv
```

Comment:
why does this file exist?

Reply:
CSVs generated here should never be commited. They are just intermediate steps for generating locales via the `manage` script.